.github/workflows/changelog_update.yml

```diff
@@ -0,0 +1,19 @@
+name: Check changelog updated
+
+on:
+  pull_request:
+    branches:
+      - master
+    types: [opened, synchronize, labeled, unlabeled]
+
+jobs:
+  check_changelog_updated:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - if: ${{ !contains(github.event.pull_request.labels.*.name, 'skip_changelog') }}
```

Comment:
Adding filter for changed files could save time here
```
- name: Filter changes
        id: changes
        uses: dorny/paths-filter@v3
        with:
          filters: |
            has_changes:
              - 'desc/**'
```
For the changelog, we don't want to include the changes in other folders anyway.

Reply:
this whole action takes a few seconds so don't think its really worth it.

Reply:
By time, I mean adding the flag. If this test runs for every PR and fails, then that small red cross could be annoying :)